modules/custom/az_mail/az_mail.install

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for az_mail module.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function az_mail_install() {
+    $config_factory = \Drupal::service('config.factory');
+    $mail_config = $config_factory->getEditable('system.mail');
+    $mail_config->set('interface.default', 'SMTPMailSystem')->save();
+}
+
+/**
+ * Force update system.mail.interface.default.
+ */
+function az_mail_update_9201() {
+    $config_factory = \Drupal::service('config.factory');
+    $mail_config = $config_factory->getEditable('system.mail');
+    $mail_config->set('interface.default', 'SMTPMailSystem')->save();
```

Comment:
Maybe we should check to make sure it's current value is the default Drupal core mail interface first?

Reply:
Also, doesn't the SMTP module save the old value in a separate config setting that it reverts to if the module is uninstalled of something?  Maybe we should also do that?

Reply:
https://git.drupalcode.org/project/mailsystem/-/blob/8.x-4.x/src/Form/AdminForm.php#L291-318